src/codeu/chat/server/Controller.java

```diff
@@ -152,6 +174,26 @@ public User login(Uuid id, String username, String password, Time creationTime)
           result);
     }
 
+    //Load in all of the user's conversations and messages
+    ArrayList<ConversationData> conversations = new ArrayList<ConversationData>();
```

Comment:
No need need to create a new `ArrayList` instance if you're going to assign it in the next statement. Combine the 2 instead.